python/tvm/relay/op/transform.py

```diff
@@ -1450,6 +1450,61 @@ def sparse_reshape(sparse_indices, prev_shape, new_shape):
     return TupleWrapper(_make.sparse_reshape(sparse_indices, prev_shape, new_shape), 2)
 
 
+def segment_sum(data, indices, num_segments=None):
+    """
+    Computes the sum along segments of a tensor. This op is much better understood with
+    visualization articulated in the following links and examples at the end of this docstring.
```

Comment:
I think it would be best if we had a mathematical description of what's happening.

Reply:
Done.